Sources/Workspace/SourceControlPackageContainer.swift

```diff
@@ -37,10 +37,6 @@ internal final class SourceControlPackageContainer: PackageContainer, CustomStri
         /// Optional suggestion for how to resolve the error.
         public let suggestion: String?
 
-        public var diagnosticLocation: DiagnosticLocation? {
-            return PackageLocation.Remote(url: self.repository.location.description, reference: self.reference)
-        }
-
```

Comment:
@neonichu is this reasonable given the information is now carried as associated error in the metadata?

Reply:
Yep, I believe we won't need the location information anymore.